### DISCUSSION TITLE
can't install

### DISCUSSION THREAD

**Discussion by georgygoshin** [mwolff44/pyfreebilling#189] [Q&A] [answered]
Hello!

I'd like to start the PKS and try to play with but installation does not really installs any containers.

root@pks:~# pks install
                                                                                                                        

  _____      _     _ _____ _______ _______     _______ ______  _______
 |_____] ___ |____/    |   |______ |______ ___ |______ |_____] |
 |           |    \_ __|__ ______| ______|     ______| |_____] |_____

   [✗] A new PKS version is available (4.0.1). Think about updating.

test if prerequesites are respected
  [✓] Detected x86_64 processor
  [✓]  Not a new install. Edit manually .env file for any changes!
  [✓]  Installation complete!

Runs very quickly but downloads nothing.

root@pks:~# cd /srv/pks/
root@pks:/srv/pks# ls -lR
.:
total 20
-rw-r--r-- 1 root root   58 Mar 27 13:59 Caddyfile
drwxr-xr-x 2 root root 4096 Mar 27 13:59 db
-rw-r--r-- 1 root root 4069 Mar 27 13:59 docker-compose.yml
drwxr-xr-x 2 root root 4096 Mar 27 13:59 redis
drwxr-xr-x 2 root root 4096 Mar 27 13:59 scripts

./db:
total 0

./redis:
total 0

./scripts:
total 20
-rwxr-xr-x 1 root root 17033 Mar 27 13:59 pks

I run Ubuntu 22.04 fresh system. Any advice how to move forward?

**Comment by mwolff44** [comment, marked as answer]
Hi,
The containers will be downloaded and launched with the command "pks start".
Do you have a .env file with the settings ?
Ubuntu 22.04 is OK !

**Reply by mwolff44** [reply]
I found the issue. In /srv/pks/scripts/pks change the line 393 to this https://github.com/mwolff44/pyfreebilling/blob/3f1a226e71e63df7593bd1b7c1687b3d96fddcc5/src/pks#L393
and launch "pks install"
I will release a new version this WE.
